DesignSafe-CI/portal
# Bug/DES-2068 & 2178: FR social sciences collection form bug & remove unsupported licenses

## Overview: ##

This PR fixes a bug when editing referenced data in social sciences collections.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2068](https://jira.tacc.utexas.edu/browse/DES-2068)
* [DES-2178](https://jira.tacc.utexas.edu/browse/DES-2178)

## Summary of Changes: ##

## Testing Steps: ##

* For 2068, edit and remove referenced data in one of the social science collections in the field recon test project. 
* For 2178, at the end of the publication preview sequence, check to see if "share alike" is still visible.

## UI Photos:

<img width="472" alt="Screen Shot 2022-02-13 at 8 37 53 PM" src="https://user-images.githubusercontent.com/47395902/153885569-421c1e50-7ee7-4d61-bc86-a189669bcd05.png">
